services/integration-content-repository/k8s/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: icr
```

Comment:
missing: declare namespace in metadata "oih-dev-ns"

Reply:
done